### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by clausmullie**

### CODE REVIEW

File: glossary.md [hunk 1]
```diff
@@ -2,11 +2,11 @@
 
 ## Code
 
-Any explicitly described system of rules is code. This includes civil code – such as laws, policy and ordinances – as well as source code that is used to build software. Both of these are rules, some executed by humans and others by machines.
+Any explicitly described system of rules. This includes civil code – such as laws, policy and ordinances – as well as source code that is used to build software. Both of these are rules, some executed by humans and others by machines.
 
 ## Codebase
 
-Any discrete package of Code – both source and policy – tests and documentation required to instantiate a piece of policy or software.
+Any discrete package of Code (both source and policy), tests and documentation required to instantiate a piece of policy or software.
```
Comment (bvhme) [inline glossary.md]: What exactly does this fix?

Comment (clausmullie) [reply glossary.md]: the comma is key for readability, no?

**Comment by bvhme** [comment]
@ElenaFdR could you take a look at this and have an opinion.

I feel the en-dashes are a good way to create a intermediate clause, but I understand it might be hard to use, perhaps commas are better across the board.

**Comment by ElenaFdR** [comment]
@bvhme I have no objection in principle to using dashes for parenthetical clauses. In thise case, the content in the em dashes functions as a parenthetical clause for the first list item. Because it's a list, I agree with @clausmullie  that the comma is important.
I've left Claus's sentence additions, but changed the sentence further.

**Pull request merged**